settings.gradle

```diff
@@ -43,7 +43,7 @@ plugins {
     id "com.gradle.enterprise" version "3.13.2"
 }
 
-rootProject.name = 'Ballerina AsyncAPI Tool'
+rootProject.name = 'asyncapi-tool'
```

Comment:
Should it be 'asyncapi-tool' or 'asyncapi-tools'? :) not sure about the convention we have followed in the openapi tool